### DISCUSSION TITLE
Possible Need of Clarification

### DISCUSSION THREAD

**Discussion by EmanuelM153** [ava-labs/avalanche-academy#81] [General]
In the _Customizing EVM_ course, on the module _05-evm-configuration_ and in the page _07-initial-token-allocation_, there is the following paragraph:
"The default configuration for testing purposes **contains configures plenty of tokens to a the address** 8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC. The private key for this address is: 56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"

Maybe, the text in bold needs some clarification or more context

**Comment by ashucoder9** [comment]
yeah, i think we can add more details on how the amount is determined (which the L1 creator ofc decides) and how a dev can check the amounts allocated. 

btw when you create the blockchain, it shows the config of how much tokens are minted to which address.
